README.md

```diff
@@ -2,13 +2,96 @@
 
 ![](Extras/img.jpg)
 
-## Python / Raspberry Pi interpretation of Bryan Boyer's Very Slow Movie Player  
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
+Very Slow Movie Player using Python + Raspberry Pi
 
-Full writeup of this project here:   
-https://medium.com/@tomwhitwell/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4
+## Background
 
-Bryan's original post here:  
-https://medium.com/s/story/very-slow-movie-player-499f76c48b62  
+In December 2018, Bryan Boyer posted [“Creating a Very Slow Movie Player”](https://medium.com/s/story/very-slow-movie-player-499f76c48b62), an essay about light and Brasília and architecture in which Boyer builds an e-paper display that shows films at 24 frames per hour, rather than 24 frames per second so it takes about a year to play the 142 minutes of 2001: A Space Odyssey.
 
+In August 2020, Tom Whitwell posted ["How to Build a Very Slow Movie Player for £120 in 2020"](https://debugger.medium.com/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4), with instructions on how to build a VSMP with the new [7.5-inch, Raspberry Pi-compatible e-paper display from Waveshare](https://www.waveshare.com/product/displays/e-paper/epaper-1/7.5inch-e-paper-hat.htm).
 
+SlowMovie is the code that runs a VSMP on your Raspberry Pi.
+
+## Install
+
+**Note:** These installation instructions assume you have access to your Raspberry Pi and that you have the hardware set up properly. See the [Medium post](https://debugger.medium.com/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4) for more complete instructions.
+
+SlowMovie requires [python3](https://www.python.org/). It uses [ffmpeg](https://ffmpeg.org/) via [ffmpeg-python](https://pypi.org/project/ffmpeg-python/) for video processing, and [Pillow](https://python-pillow.org/) for image processing.
+
+On the Raspberry Pi:
+0. Make sure SPI is turned on
+  * run `sudo raspi-config`
+  * Navigate to Interfacing Options > SPI
+1. Clone this repo
+  * `git clone https://github.com/TomWhitwell/SlowMovie/`
+  * Navigate to the new SlowMovie directory: `cd SlowMovie`
+2. Install e-paper drivers
+  * Go into the e-paper driver directory: `cd e-paper/rpi/python/`
+  * Run `sudo python3 setup.py install`
+  * Go back to /SlowMovie: `cd ../../..`
+3. Make sure dependencies are installed
+  * `sudo apt install ffmpeg`
+  * `pip3 install ffmpeg-python`
+  * `pip3 install pillow`
```

Comment:
This isn't needed anymore, I've added it and the rest of the dependencies (RPi.GPIO/Jetson.GPIO, spidev, numpy) to waveshare's setup.py so it's automatically installed.

Reply:
Here's a [link to that](https://github.com/waveshare/e-Paper/blob/master/RaspberryPi_JetsonNano/python/setup.py#L4) in the waveshare driver just to confirm. 

I'm of mixed opinion on this one as we are importing PIL as part of ```slowmovie.py``` - its not truly an external dependency that only the waveshare driver uses. If all the PIL code was contained with the EPD driver I'd say that it's that package's responsibility but we're using it too. I guess to put it another way imagine I had program `foo.py` that used library `bar`. `bar` uses `fancylib` as a dependency. That's awesome because I want to use `fancylib.do()` as part of my code too so I import `fancylib` and use it all I want. Now `bar` is an upstream library maintained by other people - they move on and no longer want to use `fancylib` any more. Now my code is broke because `fancylib` is no longer installed by the library and everyone using `foo.py` is angry it doesn't work. 

Sorry for the long hypothetical. Installing it with this one line statement doesn't cost us anything but in a future where maybe a different EPD driver is being used we'll be glad it's there since we're stilling using PIL  as part of the ```slowmovie.py``` file instead of accidentally breaking things and having to remember to add it back in.

Reply:
Okay, I get that. Since we're using it independently of EPD, we should install it ourselves.